IsraelHiking.Web/src/scss/common.scss

```diff
@@ -98,19 +98,32 @@ body {
         padding: 0 !important;
         display: block;
         width: v.$app-button-size;
-        height: v.$app-button-size;
+        height: 44px;
 
         & i {
             vertical-align: middle !important;
         }
     }
+    button.active:not(button.active:first-child , .search button.active ){
+        margin-top:7px;
+    }
 }
-
+.side-control.search
```

Comment:
Why is this different than other buttons that are part of the side control?
This should be part of the side-control definition above, shouldn't it?